code/__DEFINES/tools_defines.dm

```diff
@@ -6,6 +6,11 @@
 #define TOOL_WRENCH 		"wrench"
 #define TOOL_WELDER 		"welder"
 
+// Paperwork Tools
+#define LABEL_MODE_OFF 0
+#define LABEL_MODE_NORMAL 1
+#define LABEL_MODE_GOAL 2
```

Comment:
Unless I'm misreading something, these defines are still only just used in one file, code/modules/paperwork/handlabeler.dm, so they should continue to be defined and undefined in just that file.

Reply:
Forgot to move them back! Fixed